### ISSUE TITLE
display doesnt look good the first time you start it

### ISSUE THREAD

**Issue by ikaman** [FarGroup/FarManager#257]
After one of the newer versions of far on windows 10, fullhd, 14 inch laptop with 150%scaling I receive this 

![image](https://user-images.githubusercontent.com/1039280/88459014-423c0c00-ce92-11ea-9647-8cd663ad1c8e.png)

after i do a couple of ctrl-o and a couple of enters if fixes it self. But every time i start I get the same look

**Comment by alabuzhev** [comment]
Please report it [here](https://github.com/microsoft/terminal/issues/), it's a host issue.
I'll leave this one open in case someone finds a workaround.

**Comment by Jandini** [comment]
This happens when you set **Run: Maximized** in **Shortucut** tab. 

![image](https://user-images.githubusercontent.com/19593367/100847884-4992a800-3478-11eb-8892-c4f51436e937.png)

The issue has slightly changed in build **3.0.5700**

![image](https://user-images.githubusercontent.com/19593367/100847991-6b8c2a80-3478-11eb-9d83-030246a9aaf9.png)

I would have to check but this started happening from certain version. I do not remember when, though. 

I have got scaling set to 125%.

![image](https://user-images.githubusercontent.com/19593367/100848290-d2a9df00-3478-11eb-84f4-5a13fdff9f23.png)

**Comment by Jandini** [comment]
> Please report it [here](https://github.com/microsoft/terminal/issues/), it's a host issue.

It seems not to be related to Windows terminal. I tried running cmd.exe from a short cut with Run: Maximized and everything is all right. Also the scaling does not change anything in my case.

**Comment by ikaman** [comment]
> This happens when you set **Run: Maximized** in **Shortucut** tab.

Exactly. When running in normal mode it doesn't appear. But then again i like to run my far in maximized mode.

Is there a way to make this work in maximized mode? And I had the same experience -> it started happening after a certain version

**Comment by alabuzhev** [comment]
@ikaman can you find out after which exactly version? There are quite a few of them available on the releases page here.

**Comment by ikaman** [comment]
Every version for the last couple of months.
I have many computers on this one its v3.0.0.5577

**Comment by alabuzhev** [comment]
Please try to narrow the range.
E.g. check the version that is exactly in the middle between the one that does work and the one that doesn't, then again and so on.

**Comment by ikaman** [comment]
hi there

I have only the latest versions now and  the problem started months ago.
If I will have time I will go through a binary search of the releases and will post here

**Comment by DarkSilence** [comment]
Hi. I have a same problem on win 10. I noticed that switching to «use legacy console» fixes the issue.
When legacy console mode is off far sometimes don't resize itself when window changes size 
![image](https://user-images.githubusercontent.com/1132639/101016994-15b0a300-359c-11eb-96c4-8e60eee7713f.png)
